x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -1198,6 +1198,17 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
       }
     ),
   },
+  {
+    key: 'linux.advanced.events.aggregate',
+    first_supported_version: '8.15',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.linux.advanced.events.aggregate',
```

Comment:
We should make this `advanced.linux.advanced.events.aggregate_process` so when we add network event aggregation (or other types we haven't considered yet) users can control each independently.

Reply:
Ah, good eye. Forgot to push.